buildbot.mariadb.org/constants.py

```diff
@@ -1,3 +1,8 @@
+import yaml
+
+with open('/srv/buildbot/mariadb.org-tools/buildbot.mariadb.org/os_info.yaml', 'r') as f:
+    os_info = yaml.safe_load(f)
+
 github_status_builders = ["amd64-centos-7", "amd64-debian-10", "amd64-fedora-35", "amd64-ubuntu-2004-clang11", "amd64-windows", "amd64-windows-packages"]
 
 release_builders = ["aarch64-centos-7", "aarch64-centos-7-rpm-autobake", "aarch64-debian-9", "aarch64-debian-9-deb-autobake", "aarch64-debian-11", "aarch64-debian-11-deb-autobake", "aarch64-debian-sid", "aarch64-debian-sid-deb-autobake", "aarch64-fedora-34", "aarch64-fedora-34-rpm-autobake", "aarch64-fedora-35", "aarch64-fedora-35-rpm-autobake", "aarch64-rhel-8", "aarch64-rhel-8-rpm-autobake", "aarch64-ubuntu-1804", "aarch64-ubuntu-1804-deb-autobake", "aarch64-ubuntu-2110", "aarch64-ubuntu-2110-deb-autobake", "aarch64-ubuntu-2204", "aarch64-ubuntu-2204-deb-autobake", "amd64-debian-sid", "amd64-debian-sid-deb-autobake", "ppc64le-debian-11", "ppc64le-debian-11-deb-autobake", "ppc64le-debian-sid", "ppc64le-debian-sid-deb-autobake", "s390x-ubuntu-2004", "s390x-ubuntu-2004-deb-autobake", "s390x-ubuntu-2204", "s390x-ubuntu-2204-deb-autobake", "s390x-rhel-8", "s390x-rhel-8-rpm-autobake", "s390x-sles-15", "s390x-sles-15-rpm-autobake"]
```

Comment:
Can you put those 2 lines in list please (sort alphabetically).
Also, I probably will need a way to count the builders number, not sure how to proceed so far...

Reply:
Not sure what you want to put in list. I have sorted github_status_builders and release_builders though. Regarding counting, I think that the best way would be based on the .yaml file.

Reply:
I was speaking about something like:
```python
github_status_builders = [
    'amd64-centos-7',
    'amd64-debian-10',
    'amd64-fedora-35',
    'amd64-ubuntu-2004-clang11',
    'amd64-windows',
    'amd64-windows-packages'
]
```

I am happy to use the yaml file though, not sure how but I will find. Can you confirm that given what we have in the actual one, I should expect 7 ssh connections?

Reply:
Yes, based on the current one there should be 7 connections. If it's easier I can write the number of connections to a file after the .yaml is parsed. Is this better?